packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/list-projects-area/useProjectsProjectFilterExtensionPoints.ts

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { DataExtensionPoint } from '@eclipse-sirius/sirius-components-core';
+import { ProjectFilter } from './useProjects.types';
+
+export const useProjectsProjectFilterExtensionPoint: DataExtensionPoint<Array<ProjectFilter>> = {
```

Comment:
It should probably ask for a function to customize the GQLFilter and return it. We can't decide how to merge filters in all use cases.

Reply:
I did like `HttpOptionsConfigurer` but called it `ProjectFiltersMerger`